### ISSUE TITLE
Auto Update on Window X close?

### ISSUE THREAD

**Issue by lnolan5017** [standardnotes/forum#1299]
**Describe the issue**
I have Auto Update Enabled. If I close Standard Notes using the X in the upper right corner in Windows 10, then relaunch, it does not update. Not sure if this is intended or not. As there is no option to update from the Updates menu, it took me a while to figure out I had to go to the SN main menu and choose Window > Close then restart to get the update applied.

**To Reproduce**
1. Launch SN from the Windows 10 Taskbar
2. In the main menu, it indicates there is an update available.
3. Use the Windows close X in the upper right hand corner to close SN.
4. Launch SN from the taskbar again. 
5. The app does not update and the menu still indicates there is an update available.

**Expected behavior**
If I have auto-update enabled, it should update the first time I launched SN from the taskbar. If I need to close and relaunch through the menu, the option to do so should appear in the Updates menu through an "Update Now" option. Otherwise, there is no way to know that closing through the menu system is required. Better still would be that when I first launch SN and it detects the update, it could ask if I wanted to update now or next time I close and restart.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - SN Version 3.6.7

**Comment by Regina3000** [comment]
I am also having this issue on Windows 10, SN version 3.9.20

**Comment by atmoio** [comment]
Does your app minimize to dock on close? Or does it quit completely? Because it would have to quit completely in order for the autoupdate to trigger.

**Comment by lnolan5017** [comment]
It quits completely when I use either the window X to close or the menu
item Window > Close. I have verified that it disappears from the Task
Manager in both cases.

Here are snapshots of the settings showing "Minimize to Tray On Close" is
NOT checked and "Enable Automatic Updates" IS checked.
[image: image.png]


[image: image.png]

The update normally gets applied after I use the menu Window > Close to
exit. However, I can close multiple times with the Window X and the update
is ignored.

I could have sworn it did not update when I tried the Window > Close last
week but it worked this time.

Not sure if that helps you or not.

Thanks,
Linda
